src/modules/blockchainApplication/manage/components/RemoveApplicationSuccess/RemoveApplicationSuccess.js

```diff
@@ -14,7 +14,7 @@ const RemoveApplicationSuccess = ({ history, sharedData: { application } }) => {
         <Icon name="successCheckMark" />
         <p>{t('Application has now been removed')}</p>
         <p>
-          {t('You can always add')} <a href="#">{application.data.chainName}</a>{' '}
+          {t('You can always add')} <a href="#">{application.chainName}</a>{' '}
```

Comment:
This link makes no sense 🤔. Clicking Enevti text and be linked to my accounts :P. Would be much better to be linked to `http://0.0.0.0:8080/#/?modal=addApplicationList`

Reply:
@oskarleonard Fixed